lib/Transponder.js

```diff
@@ -417,6 +433,8 @@ class Transponder extends Emitter
 					layers		: layers
 				});
 			} else {
+				this.logger?.warn(`-Transponder::setTargetBitrate() Unable to locate layer suitable for target bitrate ${target} and strict mode, so muting`);
```

Comment:
better create a child logger with `transponder`  on constructor id and just have the method name here.

```suggestion
				this.logger?.warn(`-setTargetBitrate() Unable to locate layer suitable for target bitrate ${target} and strict mode, so muting`);
```